### PR TITLE
[add] new packages for new Data 100 HW3 and lecture demos

### LINKED ISSUE #5471: Request python package pytorch (cpu only), transformers, lxml for class Data 100
### Package Name

pytorch (cpu only), transformers, lxml

### Hub URL

data100.datahub.berkeley.edu

### Course Name

Data 100

### Semester Details

This semester and following semesters

### Installation Deadline

Ideally within 1-2 business days for content team to test out new HW3 on datahub.

### PR DESCRIPTION
**Pull request by CharlieJCJ**
Resolves #5471

### CODE REVIEW

**Comment by CharlieJCJ** [comment]
Ideally merge to staging within a day or two. Content team wants to test it out in datahub. Thank you!!!

**Comment by shaneknapp** [comment]
i'll merge it now!  give it about ~15-20m to percolate out to staging.

**Comment by CharlieJCJ** [comment]
Thank you!!

**Comment by CharlieJCJ** [comment]
@shaneknapp We have tested out the packages, except for this warning in the importing cell (which isn't an error), everything else for the homework is working fine in staging, and ready to merge to `prod`. We can tell students to ignore the warning, or hide the warning message. 

image
image

From https://stackoverflow.com/questions/65298241/what-does-this-tensorflow-message-mean-any-side-effect-was-the-installation-su
> This is not an error, it is just telling you that it can and will take advantage of your CPU to get that extra speed out.

**Comment by shaneknapp** [comment]
hey @CharlieJCJ !  we're aware of this, but since we're not compiling the package from source there's not much that we can do.
